src/components/ChecFileUploader/FileRow.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="chec-file-row" :class="{ 'chec-file-row--error': hasError }">
+    <div class="chec-file-row__left">
+      <svg
```

Comment:
Don't we have `<ChecLoading>` for this?

Reply:
Ah I think we do, but it's currently not the same. Do you think it's better to update chec-loading to support a secondary style?

Reply:
I think you should use the component that was built for that purpose, and we can update the style later.

Reply:
Using common patterns for things like this forces us to be consistent with our design implementation